.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: node_js
 node_js:
-- '8'
-- '10'
+- '12'
```

Comment:
@gmaclennan is the mobile app also using node 12 to build?

Reply:
@okdistribute the mobile app is running on Node 10 (I assume you mean the Node version it is running mapeo-core on, vs. the Node version used to build the app?). The latest `nodejs-mobile` is using Node v12.6.3 and we should be able to update to that without issues, I had only been holding off until we are sure that the stack runs ok on Node v12 (I remember that we had issues with `level` etc. not supporting Node v10 before). If we know the mapeo-core stack works with Node v12 then I can update MM.